lib/constants/branch-constants.ts

```diff
@@ -0,0 +1,8 @@
+// Branch Status
+export const BRANCH_STATUS_SUCCESS = 'success';
+export const BRANCH_STATUS_FAILURE = 'failure';
+export const BRANCH_STATUS_ERROR = 'error';
+export const BRANCH_STATUS_PENDING = 'pending';
+export const BRANCH_STATUS_CREATED = 'created';
+export const BRANCH_STATUS_RUNNING = 'running';
+export const BRANCH_STATUS_FAILED = 'failed';
```

Comment:
Please check if it's appropriate to unify failed and failure. Note: we may get back different constants *from the plastforms*, but we want to unify them into a single "branch status" internally.

Reply:
Should we perform the unification here? It will affect the application logic though.
P.S. Gitlab is using `failed` status, so I kept them as it is for now.

Reply:
OK, keep failed/failure as-is, but please create a new issue for consolidating these two into one.

Reply:
Does it need any further changes apart from this?